_pages/security/incident-response-drills.md

```diff
@@ -0,0 +1,228 @@
+1. Stop the exposure.
+  * Assess the severity and impact of the potential leak.
+  * Decide if the site needs to be set into a maintenance mode to stop further exposure. If yes, then bring up the maintenance page.
+  * If you are able to isolate the section of the site where the issue is occurring and remove/hide the page.
+1. Identify root cause of the issue and deploy a hotfix.
+1. Take necessary corrective action as directed by your agency security team. If there are corrective actions that the PO is able to handle in terms of contacting the affected users, do so.
+
+### Example drill steps:
+
+1. Acknolwedge that the first step would be to inform points of contact; establish that everyone knows who to inform in the event of an incident.
+1. In a development environment, practice putting the site into a maintenance mode or removing/hiding a page on the site, whichever would be most relevant to your project.
+1. Review any relevant corrective action / affected user notification procedures.
+
+
+## Scenario: Oops, I Deleted the Database
+
+The database needs to be restored from a backup.
+
+### Example mitigation steps:
+
+1. If you're using cloud.gov, follow [cloud.gov database backup procedures](https://cloud.gov/docs/services/relational-database/#backups).
+
+### Example drill steps:
+
+Assuming you have a staging database using a dedicated cloud.gov database plan:
+
+1. Delete some data from your staging database. (No deleting data from a production database, please.)
+2. Reach out to cloud.gov using the [the non-emergency email address provided in thir docs](https://cloud.gov/docs/services/relational-database/#backups); request a backup.
+3. Practice restoring the staging database to the point in time before you deleted the data.
+
+## Scenario: Oops, I Erased the S3 Bucket
+
+Let's re-create and restore from a backup.
+
+### Example mitigation steps:
+
+1. If the bucket no longer exists, create a new bucket in cloud.gov in the space where the bucket was deleted, ideally using infrastructure-as-code or a deploy script.
+2. Restore bucket contents from a backup.
+3. Verify the bucket settings, permissions, and contents are correct.
+
+### Example drill steps:
+
+Follow the mitigation steps above in a development environment.
+
+## Congratulations, you accidentally did compliance too!
+
+For your project, you will need an [ATO](https://before-you-ship.18f.gov/ato/). Part of that ATO is providing required documentation of controls. Controls are different security considerations. This process varies from agency to agency, so, work with your security partners to know which controls they need documented. Don't forget that you can inherit most of your [controls by using cloud.gov](https://cloud.gov/docs/security/conforming-federal-security-regulations/) and you just need to reference that it's covered. The [before you ship guide](https://before-you-ship.18f.gov/) is a great resource for ATOs.
```

Comment:
```suggestion
For your project, you will need an [ATO](https://before-you-ship.18f.gov/ato/). Part of that ATO is providing required documentation of controls. Controls are different security considerations. This process varies from agency to agency, so, work with your security partners to know which controls they need documented. Don't forget that you can inherit a substantial number of [controls by using cloud.gov](https://cloud.gov/docs/security/conforming-federal-security-regulations/) and you just need to reference that it's covered. The [Before You Ship guide](https://before-you-ship.18f.gov/) is a great resource for ATOs.
```

Reply:
Small suggestions here: use the same language that cloud.gov uses about how many controls they can provide via inheritance ("a substantial number"!). And capitalize Before You Ship. But both of these are very small things! Overall this whole section is fantastic.

Reply:
👍🏻 on this suggestion